nextcloud/spreed
# fix(call): do not update participants list often when not in the call

### ☑️ Resolves

* Hopefully reduce a server load during start of the call for large amount of participants:
  * Apply fast update only when user is in the call and has tab visible
  * Increase delay time for fetching from 3 to 5 seconds

## 🖌️ UI Checklist

### 🖼️ Screenshots / Screencasts

To test locally:
- When in chat and someone joins conversation and call within 15s - should be only one request in Networks
- When in call and someone joins conversation and call within 5s - should be only one request in Networks

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
